distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -42,6 +42,8 @@ grant codeBase "file:/catalog-core-directorymonitor/org.apache.camel.camel-core/
     // permission java.io.FilePermission "/test/some/monitored/directory${/}-", "read, write";
 }
 
+
```

Comment:
✏️ Can we remove this from the PR if it isn't changed?

Reply:
I'm trying. It feels like git is ignoring spaces. I'll get it eventually.

Reply:
I have tried everything I can think of. I even tried editing the branch by hand in github. The extra lines don't show up there.